src/pages/studies/studies.tsx

```diff
@@ -13,9 +13,12 @@ import styles from './studies.module.scss';
 let previousData: any | null = null;
 let previousMappingData: any | null = null;
 
-const Studies: FC = () => {
+const Studies = () => {
   const { filters } = useFilters();
-  let studiesResults = useGetStudiesPageData({ sqon: filters, first: 10, offset: 0 });
+  const [currentPage, setCurrentPage ] = useState(0);
+
+  let studiesResults = useGetStudiesPageData({ sqon: filters, first: 2, offset: currentPage });
+
   let studiesMappingResults = useGetExtendedMappings('studies');
 
   if (studiesResults.loading || studiesMappingResults.loadingMapping) {
```

Comment:
I'm wondering if this logic should be taken care in a non ui part ?

Reply:
this is UI concerned as well